package.json

```diff
@@ -6,8 +6,8 @@
   "author": "Symphony",
   "main": "js/main.js",
   "scripts": {
-    "dev": "npm run browserify-preload && cross-env ELECTRON_DEV=true electron .",
-    "demo": "npm run browserify-preload && cross-env ELECTRON_DEV=true electron . --url=file://$(pwd)/demo/index.html",
+    "dev": "npm run browserify-preload && cross-env ELECTRON_DEV=true electron . --enable-sandbox",
```

Comment:
You'd think so, but it doesn't. See the comment in the above discussion from the electron contributor who added sandbox support:

> Passing sandbox alone does not enable chromium os sandbox, it simply creates a renderer that is compatible with it.